assets/templates/metaboxes/metabox.error.php.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
@christianwach is this file meant to be named `metabox.error.php.php`?

Reply:
@kcristiano I know it looks weird but, yes - the templates are named `metabox.error.[function-of-template].php`. This one happens to be the one that handles the PHP version.